report/thesis.tex

```diff
@@ -1035,7 +1030,7 @@ \chapter{Conclusion}
 
```

Comment:
"Decentralized fork accountability" 
I wouldn't call this problem "Decentralized fork accountability" since we have that (if we mention that what monitor does is done by every correct process".
Instead, I think Adi's proposal is great "Fork tolerance".
Since the goal of this problem is to eject faulty processes and eventually reach the decision.

Reply:
I agree, modified the title and the first part of the section accordingly.